contracts/pools/SyndicatePoolBase.sol

```diff
@@ -309,7 +309,7 @@ abstract contract SyndicatePoolBase is IPool, SyndicateAware, ReentrancyGuard {
     uint256 _amount,
     uint64 _lockUntil,
     bool _useSSYN
-  ) external override {
+  ) external override poolAlive {
```

Comment:
Do we need to check poolAlive also in the internal functions? If they are called by the external function and in the external functions the modifier is used, it is just going to cause more computation and increase the gas cost.

Alternatively, we can put it ONLY in the internal functions.